smartseq2_single_sample/ss2_portability_inputs.json

```diff
@@ -0,0 +1,17 @@
+{
+  "SS2PortabilityWorkflow.gtf_file": "https://s3.amazonaws.com/mk-hca-portability-data/new_ss2/gencode.v27.primary_assembly.annotation.gtf",
```

Comment:
Should we use the input files that are in the mint team google cloud storage instead?

Reply:
Longterm storage I would suggest using the Prod Blue box, until then I imagine solutions are temporary.

Reply:
Yeah, we need a publicly-accessible place to store the test data. Right now, I believe all the Mint google buckets are private, so they won't work. The prod blue box makes sense (though how do we ingest that?). In the meantime, it would be great to have a public place in google cloud storage where we could put all of our test data.